lib/demos/material/top_app_bar_demo.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';
+
+// BEGIN topAppbarDemo
+
+class TopAppBarDemo extends StatelessWidget {
+  const TopAppBarDemo({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    var localization = GalleryLocalizations.of(context);
+    return Scaffold(
+      appBar: AppBar(
+        leading: const Icon(
+          Icons.tag_faces,
```

Comment:
Each IconButton needs a semanticLabel for accessibility. I think you should use the same icons in this demo as the `BottomAppBar` demo so that we do not need additional translations.